src/phases/filesystem/FlushingTemplate.yml

```diff
@@ -0,0 +1,1656 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This file defines a template to use to stress checkpointing, journal flush and dirty memory. The test itself is
+  simple:
+    - Run a single CrudActor targeting:
+      - test database, configurable via the Database parameter.
+      - Collection0 collection, configurable via the Collection parameter.
+      - 10 threads, configurable via the Threads parameter.
+      - 5000 iterations, configurable via the Iterations parameter.
+      - 1500 documents, configurable via the Documents parameter.
+      - 1 x 1000 bytes field in document, configurable via the DocumentSize parameter.
+      - {w:1} insert options, configurable via the Options parameter.
+  
+
+GlobalDefaults:
+  - rdIdExtraPaddingParam: &RecordIdExtraPaddingParam {^Parameter: {Name: "RecordIdExtraPadding", Default: 0}}
+  - PaddingGenerator: &PaddingGenerator {^FastRandomString: {length: *RecordIdExtraPaddingParam, alphabet: "a"}}
```

Comment:
I feel like we could make this more efficient for the default case of making strings of a. It wouldnt' keep the generality though.